vkconfig/mainwindow.cpp

```diff
@@ -78,6 +78,9 @@ MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent), ui_(new Ui::MainW
     ///////////////////////////////////////////////
     Configurator &configurator = Configurator::Get();
 
+    if (!configurator.VK_LAYER_PATH.isEmpty())
+        ui_->groupBoxProfiles->setTitle("Vulkan Layers Configurations With VK_LAYER_PATH Precedence");
```

Comment:
Is this the right place for this? I would expect that layers will come from various place, so it should be a per layer information. Probably for the layer window and the console log at start up

Reply:
Karl's request was that it be somewhere really up front and obvious. It's not likely to be in effect much on Windows, and almost never on macOS. Just on Linux when someone is "tinkering", so I think it's okay.